### PR TITLE
Update edc extended meeting due to change of invitation

### PR DESCRIPTION
**Pull request by lgblaumeiser**
## Description

As the current inviting person leaves the project, the meeting has been rescheduled. Due to that, the information for the open meeting reference needs to be updated

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by lgblaumeiser** [comment]
@stephanbcbauer : Hi Stephan, please check this pull request, we had to update the open meetings, because Ben is leaving the project and I had to reschedule the meetings. This means new dial in information that should be reflected on the page. Thanks!

File: static/meetings/edc-weekly-extended.ics [hunk 1]
```diff
@@ -19,1488 +19,328 @@ TZOFFSETTO:+0200
 END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VEVENT
+ATTENDEE;CN="Westphal, Benjamin BW. (415)";RSVP=FALSE:mailto:benjamin.bw.we
```
Comment (stephanbcbauer) [inline static/meetings/edc-weekly-extended.ics:22]: Currently, the attendees are also included in the calendar file. As I remember from last PRs... we discussed deleting them.

**Review by stephanbcbauer — CHANGES_REQUESTED** [review]
The calendar file acts just a blocker. In previous PRs there was a discussion to not showing the attendees here. They can be deleted. thx

File: static/meetings/edc-weekly-extended.ics (hunk 1 above)
Comment (lgblaumeiser) [reply static/meetings/edc-weekly-extended.ics]: Good point, I have removed the attendees from the file

**Comment by lgblaumeiser** [comment]
@stephanbcbauer @carslen @SebastianBezold @FaGru3n : Would be great if you could check the PR, it would be nice to have the reference in place, so that people do not end up in the old meeting. Thanks!

**Review by carslen — APPROVED** [review]
LGTM

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Pull request merged**